static/js/components/SiteCollaboratorList.tsx

```diff
@@ -64,75 +72,78 @@ export default function SiteCollaboratorList(): JSX.Element | null {
-                            <i className="material-icons">edit</i>
-                          </Link>
-                          <i
-                            className="material-icons"
-                            onClick={() => {
-                              setSelectedCollaborator(collaborator)
-                              toggleDeleteModal()
-                            }}
-                          >
-                            delete_outline
-                          </i>
-                        </>
-                      ) : null}
-                    </td>
-                  </tr>
-                )
-              )}
-            </tbody>
-          </table>
-        </div>
-      </Card>
-      <Dialog
-        open={deleteModal}
-        toggleModal={toggleDeleteModal}
-        headerContent={"Remove collaborator"}
-        bodyContent={`Are you sure you want to remove ${
-          selectedCollaborator ? selectedCollaborator.name : "this user"
-        }?`}
-        acceptText="Delete"
-        onAccept={() => {
-          onDelete()
-          toggleDeleteModal()
-        }}
-      ></Dialog>
-    </div>
+    <>
+      <SiteCollaboratorDrawer
+        siteName={name}
+        collaborator={selectedCollaborator}
+        visibility={editVisibility}
+        toggleVisibility={toggleEditVisibility}
+      />
+      <div className="collaborator-list">
+        <Card>
+          <div className="d-flex justify-content-between align-items-center pb-3">
+            <h3>Collaborators</h3>
+            <a
+              className="collaborator-add-btn btn blue-button"
+              onClick={startEdit(null)}
+            >
+              Add collaborator
+            </a>
+          </div>
+          <div className="narrow-page-body pb-5">
+            <table className="table collaborator-table">
+              <tbody>
+                {collaborators.map(
+                  (collaborator: WebsiteCollaborator, i: number) => (
+                    <tr key={i}>
+                      <td className="pr-5">
+                        {collaborator.name || collaborator.email}
+                      </td>
+                      <td className="pr-5 gray">
+                        {ROLE_LABELS[collaborator.role]}&nbsp;
+                      </td>
+                      <td className="gray">
+                        {EDITABLE_ROLES.includes(collaborator.role) ? (
+                          <>
+                            <a
+                              className="edit-link"
+                              onClick={startEdit(collaborator)}
+                            >
+                              <i className="material-icons">edit</i>
+                            </a>
+                            <i
+                              className="material-icons"
+                              onClick={() => {
+                                setSelectedCollaborator(collaborator)
+                                toggleDeleteModal()
```

Comment:
I know this is existing code, but we have the `startEdit` helper method for running all the actions related to editing a collaborator. Seems appropriate that we'd have the same for deleting